dftparse/vasp/tests/test_outcar_parser.py

```diff
@@ -0,0 +1,21 @@
+from dftparse.vasp.outcar_parser import OutcarParser
+
+
+def _flatten(ds):
+    """Helper to flatten a list of dictionaries"""
+    res = {}
+    [res.update(d) for d in ds]
+    return res
+
+
+def test_parse_magnetization():
+    """Test that the total magnetization parses out correctly"""
+    lines = """
+     eigenvalue-minimisations  :  9897
+ total energy-change (2. order) :-0.2503977E-01  (-0.2503131E-01)
+ number of electron      12.9999995 magnetization      -0.0000036
+ augmentation part       12.9999995 magnetization      -0.0000036
+    """.split("\n")
+    res = _flatten(OutcarParser().parse(lines))
+    assert res["total magnetization"] == -0.0000036, "Parsed the total magnetization incorrectly"
```

Comment:
Is python smart enough with precision for straight float equalities to be safe?

Reply:
In general, no.  But in this case, we're using the consistency of `float("-0.0000036")` and the interpreter's `eval(-0.0000036)`, which should be rounded in the same way (I think).  The tests pass, so :man_shrugging: